lib/infrastructure/daos/llenado_dao.dart

```diff
@@ -151,19 +174,20 @@ class LlenadoDao extends DatabaseAccessor<MoorDatabase> with _$LlenadoDaoMixin {
             u.preguntaId.equals(pregunta.id) &
             u.inspeccionId.equals(inspeccionId),
       );
-    final res = await query.get();
-
-    ///Si no han contestado la pregunta se envia nulo para que el control cree una por defecto
-    if (res.isEmpty) return null;
-
-    final respuesta = res.last;
-    return respuesta.toCompanion(true);
+    //Se supone que solo debe haber una respuesta por [pregunta] numerica para [inspeccionId]
+    final res = await query.getSingleOrNull();
+    if (res != null) {
+      return bl_dom.RespuestaNumerica(getMetaRespuesta(res),
+          respuestaNumerica: res.valor);
+    }
+    return null;
```

Comment:
Usar una condición de guarda para que quede mas bonito
```suggestion
    if (res == null) return null;
    return bl_dom.RespuestaNumerica(getMetaRespuesta(res),
          respuestaNumerica: res.valor);
    
```